portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I do some photograph', 'I do some videography', 'I have coded in 8+ different languges', 'I am from Mobile Alabama'];
```

Comment:
let's maintain single responsibility for functions. I believe we can declare consts outside.